examples/cpp/flow_control/SubscriberApp.hpp

```diff
@@ -86,11 +92,16 @@ class SubscriberApp : public Application,  public DataReaderListener
     mutable std::mutex terminate_cv_mtx_;
 
     std::condition_variable terminate_cv_;
+
+    eprosima::fastdds::rtps::EntityId_t slow_writer_guid;
+
+    eprosima::fastdds::rtps::EntityId_t fast_writer_guid;
```

Comment:
What happens if I run several publisher instances?

Reply:
Right! Before it overwrote the writers, so now I have handled also this case